src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -82,8 +83,9 @@ export const View = (props: Props) => {
     },
   ];
   const breachesDataArray = props.userBreaches.flat();
-  const scanInProgress =
-    props.userScanData.scan?.onerep_scan_status === "in_progress";
+  const initialScanInProgress =
+    props.userScanData.scan?.onerep_scan_status === "in_progress" &&
+    props.scanCount === 1;
```

Comment:
Would this never be 0?

Reply:
This can be `0`, but if it is that’s the case there would be also no scan in progress.